docs/content/concepts/configuration/config-schema.mdx

```diff
@@ -15,69 +15,75 @@ Run configuration allows providing parameters to jobs at the time they're execut
 
 Dagster includes a system for gradually-typed configuration schemas. These make it easy to catch configuration errors before job execution, as well as to learn what configuration is required to execute a job.
 
-## Using Configuration Inside an Op
+## Defining and Accessing Configuration for an Op, Asset, or Resource
 
-This example shows how to write an op whose behavior is based on values that are passed in via configuration:
+Configurable parameters accepted by an op, asset, or resource are specified by providing a `config_schema` to the corresponding decorator. The structure of a `config_schema` is very flexible and [fully documented in the API Reference](/\_apidocs/config). However, most of the time you will want to provide a Python dictionary, with keys the names of parameters and values the types of those parameters.
 
-```python file=/concepts/configuration/configurable_op.py
-from dagster import job, op
+During execution, the specified parameters are accessible within the body of the op/asset/resource under `context.op_config` (for ops/assets) or `context.resource_config` (for resources). It might seem confusing that asset config is accessed under `context.op_config` instead of `context.asset_config`. However, assets are wrappers for ops, so when we access asset config we are literally just accessing config for the underlying op.
 
+Below we define a simple op and asset with identical `config_schemas` defining a single configurable parameter, `person_name`, as well as a resource with a configurable `url` parameter:
 
-@op
-def uses_config(context):
-    for _ in range(context.op_config["iterations"]):
-        context.log.info("hello")
+```python file=/concepts/configuration/configurable_op_asset_resource.py startafter=start endbefore=end
+@op(config_schema={"person_name": str})
+def op_using_config(context):
+    return f'hello {context.op_config["person_name"]}'
 
 
-@job
-def config_example():
-    uses_config()
-```
+@asset(config_schema={"person_name": str})
+def asset_using_config(context):
+    # Note how asset config is also accessed with context.op_config
+    return f'hello {context.op_config["person_name"]}'
 
-## Providing Run Configuration
 
-How you specify config values depends on how you're running your job:
+@resource(config_schema={"url": str})
+def resource_using_config(context):
+    return MyDatabaseConnection(context.resource_config["url"])
+```
 
-### Python API
+<Note>
+  It is technically possible to access `context.op_config` inside ops (not
+  assets) without defining a `config_schema`. However, this is not recommended
+  and will likely be deprecated in the future.
+</Note>
 
-When executing a job with <PyObject object="JobDefinition" method="execute_in_process"/>, you can specify the config values through `run_config` argument:
+You can also build config into jobs, as described in [this section of the Jobs concept page](/concepts/ops-jobs-graphs/jobs-graphs#advanced-job-configuration).
```

Comment:
```suggestion
You can also build config into jobs, as described in [the Jobs documentation](/concepts/ops-jobs-graphs/jobs-graphs#advanced-job-configuration).
```

Sorry, nit here too. I'm trying to make internal references a bit more consistent and concise.